README.md

```diff
@@ -1,12 +1,36 @@
-# Overview
+# Python bindings for biscuit
 
-This is a _very_ experimental take on Python bindings for the [biscuit_auth](https://docs.rs/biscuit-auth/latest/biscuit_auth/) Rust library. It is very much a work in progress (limited testing, most errors are dropped on the floor, etc). Please don't use this for anything in its current state.
+This library provides python bindings to the [biscuit_auth](https://docs.rs/biscuit-auth/latest/biscuit_auth/) rust library.
 
-Hopefully someday this leads to a production-ready Biscuit library for Python, but this is (currently) not that, and there will definitely be significant API changes before that happens.
+It is `0.x` quality, so you can expect some API changes, even though most of the use cases are covered:
```

Comment:
The use of "0.x quality" sounds kind of negative imo (but then again, maybe it's because im not familiar with it?). Anyway, maybe we could reformulate?
```suggestion
As it is a pre-1.0 version, you can expect some API changes. However, most of the use cases are covered:
```